staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -181,6 +182,11 @@ func (c *RESTClient) Patch(pt types.PatchType) *Request {
 	return c.Verb("PATCH").SetHeader("Content-Type", string(pt))
 }
 
+// Apply begins a PATCH request with the apply patch type.
+func (c *RESTClient) Apply() *Request {
```

Comment:
Alternately, you could take some parameters like "Force" and "Manager", what do you think?

Reply:
Im adding them now. That was a good idea. There are also more options it might make sense to add but it would make the parameters messy

Reply:
On second thought, the manager does not **look** easy to add based on the docs [here](https://kubernetes.io/docs/reference/using-api/api-concepts/).

Is there a better reference to modify the manager in the request?

Reply:
The manager should be as easy to add as the force flag, it's just a query parameter.

Reply:
is it called `manager`? In the doc about apply it looks like it is a part of the body of the request

Reply:
It's called `fieldManager`